docs/tutorials/autogluon-cloud.md

```diff
@@ -1,108 +1,184 @@
 ```
 
 
 ## Installation
 `autogluon.cloud` does not come with the default `autogluon` installation. You can install it via:
 
 ```bash
-pip3 install autogluon.cloud
+pip install autogluon.cloud
 ```
 
 Also ensure that the latest version of sagemaker python API is installed via:
 
 ```bash
-pip3 install -U sagemaker
+pip install -U sagemaker
 ```
 
 This is required to ensure the information about newly released containers is available.
 
-## Prepare an AWS Role with Necessary Permissions
-`autogluon.cloud` utilizes various AWS resources to operate.
-To help you to setup the necessary permissions, you can generate trust relationship and IAM policy with our utils through
+## Prepare an IAM Role with Necessary Permissions
+Currently, AutoGluon-Cloud can use two cloud backends: **Amazon SageMaker** and **Ray (AWS)**.
+Here is an overview of the features supported by each backend.
 
-```python
-from autogluon.cloud import TabularCloudPredictor  # Can be other CloudPredictor as well
+| Feature                        | SageMaker | Ray (AWS) |
+|--------------------------------|---------------|--------------|
+| **Supported modalities**       | `tabular`, `timeseries`, `multimodal` | `tabular` |
+| **Training (single instance)** | ✅ | ✅ |
+| **Training (distributed)**     | ❌ | ✅ |
+| **Inference endpoints**        | ✅ | ❌ |
+| **Batch inference**            | ✅ | ❌ |
 
-TabularCloudPredictor.generate_default_permission(
-    backend="BACKNED_YOU_WANT",  # We currently support "sagemaker" and "ray_aws"
-    account_id="YOUR_ACCOUNT_ID",  # The AWS account ID you plan to use for CloudPredictor.
-    cloud_output_bucket="S3_BUCKET"  # S3 bucket name where intermediate artifacts will be uploaded and trained models should be saved. You need to create this bucket beforehand.
-)
-```
+AutoGluon-Cloud needs to interact with various AWS resources. For this purpose, we recommend to set up a dedicated IAM role with the necessary permissions. This can be done using one of the following options.
 
-```{note}
-Make sure you review the trust relationship and IAM policy files, and make necessary changes according to your use case before applying them.
-```
+::::{tab-set}
+:::{tab-item} CloudFormation (AWS CLI)
+:sync: setup-cli
+1. Download and review the CloudFormation template from the [AutoGluon-Cloud repository](https://github.com/autogluon/autogluon-cloud/tree/add-cfn-templates/cloudformation)
+    ```bash
+    BACKEND="sagemaker"  # Supported options "sagemaker", "ray_aws"
+    wget https://raw.githubusercontent.com/autogluon/autogluon-cloud/refs/heads/add-cfn-templates/cloudformation/ag_cloud_$BACKEND.yaml
+    ```
+    ```{note}
+    Make sure you review the IAM policy defined in the CloudFormation template, and make necessary changes according to your use case before applying it.
+    ```
```

Comment:
In theory, we could remove this step and have a "1-click-setup" experience by hosting the CloudFormation templates on S3 instead of GitHub. In that case, we could directly use the S3 URL in `create-stack`, skipping the extra step for downloading the template
```bash
aws cloudformation create-stack \
    --stack-name ag-cloud \  
    --template-body https://autogluon.s3.amazonaws.com/cloudformation/ag_cloud_sagemaker.yaml \
    --capabilities CAPABILITY_NAMED_IAM  # give permission to create IAM roles
```
However, in this case versioning will be tricky, and the user won't have an opportunity to review the template before deploying the stack.

Reply:
I think keeping the file on GitHub might be easier to maintain and versioning. Users might often need to alter configurations in the template anyways, so they'll have to download and use a local file for `template-body`.